src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3040,6 +3040,40 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <ReferenceComWrappersToCopyLocal Include="@(ComReferenceWrappers)" Condition="'%(ComReferenceWrappers.CopyLocal)'!='false'"/>
     </ItemGroup>
 
+  </Target>
+
+    <!--
+    ============================================================
+                                        _GetAssembliesMetadata
+
+    Resolve Assembly attributes for assemblies
+    The ResolveComReferences dependency is needed for defining COM-referenced wrappers only,
+    but this target handles all the assembly references from ResolveAssemblyReferences
```

Comment:
Could you depend on `ResolveReferences`? I don't understand why one source of references is explicitly `DependsOn` but the other is not.

Reply:
From my understanding, ResolveComReferences defines wrappers for com objects (paths to the assemblies) + ResolveReferences is defined as dependency for the target.